tests/backend/pytorch_backend/test_torch_ops.py

```diff
@@ -37,3 +45,44 @@ def test_unload_from_worker(self, model, mock_worker):
 
         assert not hasattr(mock_worker, "torch_model")
         assert not hasattr(mock_worker, "cfg")
+
+
+class DummyModel(torch.nn.Module):
```

Comment:
Good test, let's rename the model to `DummyModelWithDictOutput` or something less verbose but still clear what we're testing here

Reply:
DONE